Makefile

```diff
@@ -1,5 +1,5 @@
 SDK ?= "iphonesimulator"
-DESTINATION ?= "platform=iOS Simulator,name=iPhone X"
+DESTINATION ?= "platform=iOS Simulator,name=iPhone 8"
```

Comment:
Guessing this has to do with the simulators available on CI?

Reply:
Yeah also in xcode, iphone X is no longer a supported simulator device